inc/saiport.h

```diff
@@ -368,6 +368,17 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_SCHEDULER_GROUP_LIST,
 
+    /**
+     * @brief The maximum accumulative headroom size of a port in bytes
```

Comment:
1) This line should be reworded to make it easier to understand. Like:

```
The sum of the headroom size of the ingress priority groups belonging to this port should not exceed SAI_PORT_ATTR_QOS_MAXIMUM_HEADROOM_SIZE value.
```

Reply:
ok. Fixed.